bucket/everything.json

```diff
@@ -1,6 +1,6 @@
 {
     "version": "1.4.1.987",
-    "description": "Locate files and folders by name instantly.",
+    "description": "Locate files and folders by name instantly",
```

Comment:
Why do you fix something which is not broken??

```suggestion
    "description": "Locate files and folders by name instantly.",
```

![image](https://user-images.githubusercontent.com/13260377/88949779-fc1ce900-d293-11ea-9c9b-9e0f583b855a.png)

Reply:
Because in one of my previous PRs you literally just suggested to remove the dot at the end of description.

Reply:
Because it was not valid English sentence.